dependencies.yaml

```diff
@@ -136,8 +150,6 @@ dependencies:
           - matrix: {cuda: "11.*"}
             packages:
               - libucx-cu11==1.15.0
-          # NOTE: this fallback needs to be a real, suffixed version
-          #       so 'pip install .' (e.g. as used in docs builds) will work
```

Comment:
I accidentally left behind this comment in #1041 ... it's not accurate on `branch-0.39` or on this branch.